### ISSUE TITLE
Compilation fails at stdlib.h when including memory.h

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vgmurphy/limbo-android#7]
```
What steps will reproduce the problem?
1. Run make as in the instructions in the README file.
2. Compilation fails.
3.

What is the expected output? What do you see instead?
Should compile without a warning instead it sources memory.h in the qemu 
directory.

Please use labels and text to provide additional information.


```

Original issue reported on code.google.com by `ideal2i...@gmail.com` on 4 Oct 2012 at 1:59

**Comment by GoogleCodeExporter** [comment]
```
Workaround:
Comment out line in stdlib.h temporarily and recompile (I know it's ugly but 
works for now)
//#include memory.h
```

Original comment by `ideal2i...@gmail.com` on 4 Oct 2012 at 2:02

**Comment by GoogleCodeExporter** [comment]
Original comment by `ideal2i...@gmail.com` on 6 Oct 2012 at 8:25
- Changed title: **Compilation fails at stdlib.h when including memory.h**

**Comment by GoogleCodeExporter** [comment]
```
This issue was resolved in one of the last merges.
The trunk now has a 'include-fixed' library that contains a fixed version of 
stdlib.

So, this issue can be marked as resolved.
```

Original comment by `luna.sta...@gmail.com` on 26 Oct 2012 at 9:38

**Comment by GoogleCodeExporter** [comment]
```
Patch approved, fix will be included in rel 0.9.3.
```

Original comment by `ideal2i...@gmail.com` on 28 Oct 2012 at 7:36
- Changed state: **Fixed**
